scripts/build-chains.js

```diff
@@ -158,8 +158,9 @@ function findFileByTicker(tickers, dirPath) {
 async function saveNewFile(newJson, file_name) {
   try {
     const filePath = path.resolve(CONFIG_PATH, file_name);
-
-    await writeFile(filePath, JSON.stringify(newJson, null, 2));
+    const jsonContent = JSON.stringify(newJson, null, 2);
+    const contentWithNewline = jsonContent + '\n';
```

Comment:
Could you explain what happened with the new line, please?

Reply:
I've added it in order to compliance POSIX standard:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795